cmd/worker/main.go

```diff
@@ -133,16 +134,34 @@ func handleMessage(ctx context.Context, msg *pubsub.Message, packagesBucket *blo
 		}
 	}
 
+	if notificationTopic != nil {
+		err := notification.PublishAnalysisCompletion(ctx, notificationTopic, name, version, ecosystem)
```

Comment:
Style nit: in this kind of situation you can do `if err := ... ; err != nil { return err }

Reply:
That's super handy to know! I'm concerned the line would get too long by doing that and breaking it up over multiple lines seems more consistent with the lines immediately before it. I can change it if you still think it's better on one line in this case

Reply:
That's true, I hadn't seen the lines above. You're right, it's better to keep them consistent, so let's keep it as is :)